ui/page/transaction/transaction_details_page.go

```diff
@@ -695,6 +730,29 @@ func (pg *TxDetailsPage) Handle() {
 			pg.ChangeFragment(NewTransactionDetailsPage(pg.Load, pg.ticketSpent))
 		}
 	}
+
+	if pg.rebroadcastClickable.Clicked() {
+		if pg.unmined {
+			if !pg.Load.WL.MultiWallet.IsConnectedToDecredNetwork() {
+				// if user is not conected to the network, notify the user
+				pg.Toast.NotifyError("Not connected to the decred network")
+				return
+			}
+
+			err := pg.wallet.PublishUnminedTransactions()
+			if err != nil {
+				// If transactions are not published, notify the user
+				pg.Toast.NotifyError(err.Error())
+			} else {
+				pg.Toast.NotifyError("Republished unmined transactions to the decred network")
+			}
+		}
+	}
+
+	if pg.transaction.BlockHeight > -1 {
+		pg.unmined = false
+		pg.rebroadcastClickable.SetEnabled(false, nil)
+	}
```

Comment:
this isn't needed as the button won't be visible if the height is greater than -1